Travel Scheduler/View Controllers/FirstScreenViewController.m

```diff
@@ -88,6 +101,61 @@ - (void)viewDidLoad {
     [self.view addSubview:self.placesSearchBar];
     [self createLabels];
     [self createButton];
+    [self createFilterForGMSAutocomplete];
+    [self createAutocompleteTableView];
+    self.autocompleteTableView.delegate = self;
+    self.autocompleteTableView.dataSource = self;
+}
+
+#pragma mark - GMSAutocompleteFetcherDelegate
+
+NSMutableArray *resultsArr;
```

Comment:
Why is this here, rather than as a top-level declaration?

Reply:
Moved to the top as a property